currency_rate_update/README.rst

```diff
@@ -1,67 +1,92 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
-   :alt: License: AGPL-3
-
 ====================
 Currency Rate Update
 ====================
 
-Base module to download exchange rates automatically from the Internet.
-
-This module download exchange rates automatically from European central bank service (ported by Grzegorz Grzelak - OpenGLOBE.pl)
-The reference rates are based on the regular daily query procedure between central banks within and outside the European System of Central Banks, which normally takes place at 2.15 p.m. (14:15) ECB time. Source in EUR. http://www.ecb.europa.eu/stats/exchange/eurofxref/html/index.en.html
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fcurrency-lightgray.png?logo=github
+    :target: https://github.com/OCA/currency/tree/12.0/currency_rate_update
+    :alt: OCA/currency
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/currency-12-0/currency-12-0-currency_rate_update
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/259/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module provides base for building exchange rates providers and bundles
+following built-in providers:
+
+ * **European Central Bank** (ported by Grzegorz Grzelak - OpenGLOBE.pl):
+   reference rates are based on the regular daily query procedure between
+   central banks within and outside the European System of Central Banks,
+   which normally takes place at 2:15 p.m. (14:15) ECB time. Source is in
+   EUR, for more details see `corresponding ECB page <https://www.ecb.europa.eu/stats/policy_and_exchange_rates/euro_reference_exchange_rates/html/index.en.html>`_.
+
+This module is compatible with ``currency_rate_inverted`` module provided by
```

Comment:
Isn't inverted already supported here?

Reply:
The actual _usage_ of the inverted rate - no, it _stores_ the inverted rate if there's one and there's a module installed and this currency is inverted: https://github.com/brainbeanapps/currency/blob/e609c9d946ce93ee90385ad5e6ae6b5eb10bb4ea/currency_rate_update/models/res_currency_rate_provider.py#L248